cmake results in errors by lack of a CMakeLists.txt in the examples directory.

```bash
mkdir build
pushd build
cmake -D CMAKE_INSTALL_PREFIX=${HOME}/py3-caffe-refinedet -D python_version=3 -D CPU_ONLY=1 ..
...
-- Found the following Boost libraries:
--   python-py36
-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE)
CMake Error at CMakeLists.txt:74 (add_subdirectory):
  The source directory

    /home/k-tanabe/RefineDet/examples

  does not contain a CMakeLists.txt file.


-- Found Git: /usr/bin/git (found version "2.17.1")
--
-- ******************* Caffe Configuration Summary *******************
-- General:
--   Version           :   1.0.0-rc3
--   Git               :   52b6fe2
--   System            :   Linux
--   C++ compiler      :   /usr/bin/c++
--   Release CXX flags :   -O3 -DNDEBUG -fPIC -Wall -Wno-sign-compare -Wno-uninitialized
--   Debug CXX flags   :   -g -fPIC -Wall -Wno-sign-compare -Wno-uninitialized
--   Build type        :   Release
--
--   BUILD_SHARED_LIBS :   ON
--   BUILD_python      :   ON
--   BUILD_matlab      :   OFF
--   BUILD_docs        :   ON
--   CPU_ONLY          :   1
--   USE_OPENCV        :   ON
--   USE_LEVELDB       :   ON
--   USE_LMDB          :   ON
--   ALLOW_LMDB_NOLOCK :   OFF
--
-- Dependencies:
--   BLAS              :   Yes (Atlas)
--   Boost             :   Yes (ver. 1.65)
--   glog              :   Yes
--   gflags            :   Yes
--   protobuf          :   Yes (ver. 3.0.0)
--   lmdb              :   Yes (ver. 0.9.21)
--   LevelDB           :   Yes (ver. 1.20)
--   Snappy            :   Yes (ver. ..)
--   OpenCV            :   Yes (ver. 3.2.0)
--   CUDA              :   No
--
-- Python:
--   Interpreter       :   /usr/bin/python3 (ver. 3.6.9)
--   Libraries         :   /usr/lib/x86_64-linux-gnu/libpython3.6m.so (ver 3.6.9)
--   NumPy             :   /home/k-tanabe/.local/lib/python3.6/site-packages/numpy/core/include (ver 1.17.3)
--
-- Documentaion:
--   Doxygen           :   No
--   config_file       :
--
-- Install:
--   Install path      :   /home/k-tanabe/py3-caffe-refinedet
--
-- Configuring incomplete, errors occurred!
See also "/home/k-tanabe/RefineDet/build/CMakeFiles/CMakeOutput.log".
See also "/home/k-tanabe/RefineDet/build/CMakeFiles/CMakeError.log".
```

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -71,7 +71,7 @@ include_directories(BEFORE src) # This is needed for gtest.
 add_subdirectory(src/gtest)
 add_subdirectory(src/caffe)
 add_subdirectory(tools)
-add_subdirectory(examples)
+# add_subdirectory(examples)
 add_subdirectory(python)
 add_subdirectory(matlab)
 add_subdirectory(docs)

